installer/templates/phx_single/lib/app_name_web.ex

```diff
@@ -41,9 +41,10 @@ defmodule <%= @web_namespace %> do
       use Phoenix.Controller,
         formats: [:html, :json],
         layouts: [html: <%= @web_namespace %>.Layouts]
+      <%= if @gettext do %>
```

Comment:
The new lines are tricky to get right, but I am almost sure this is what is needed. You have to replicate this below.

```suggestion
        layouts: [html: <%= @web_namespace %>.Layouts]<%= if @gettext do %>

```

Reply:
With `--gettext`:

```elixir
    quote do
      # Translation
      use Gettext, backend: GtestWeb.Gettext
      
      # HTML escaping functionality
      import Phoenix.HTML
      # Core UI components
      import GtestWeb.CoreComponents

      # Shortcut for generating JS commands
      alias Phoenix.LiveView.JS

      # Routes generation with the ~p sigil
      unquote(verified_routes())
    end
```

With `--no-gettext`:

```elixir
    quote do
      # HTML escaping functionality
      import Phoenix.HTML
      # Core UI components
      import GtestWeb.CoreComponents

      # Shortcut for generating JS commands
      alias Phoenix.LiveView.JS

      # Routes generation with the ~p sigil
      unquote(verified_routes())
    end
```

I think we're good? I would love to keep the `use` above `import` as it seems like it's a pretty common ordering. 🙃

Reply:
(this is with changes I’m about to push to be clear!)